### ISSUE TITLE
Hi, I am doing a cell detection project. I am curious that if that possible to label a cell with some certainty? for example, a cell could be labelled as "60%" for type A?

### ISSUE THREAD

**Issue by WilburZjh** [AlexeyAB/darknet#6673]
If something doesn’t work for you, then show 2 screenshots:
1. screenshots of your issue
2. screenshots with such information
```
./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/dog.jpg
 CUDA-version: 10000 (10000), cuDNN: 7.4.2, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2070
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
   0 conv     32       3 x 3/ 1    608 x 608 x   3 ->  608 x 608 x  32 0.639 BF
```

If you do not get an answer for a long time, try to find the answer among Issues with a Solved label: https://github.com/AlexeyAB/darknet/issues?q=is%3Aopen+is%3Aissue+label%3ASolved

**Comment by stephanecharette** [comment]
Here is an example of cancer cell detection done using YOLOv3-tiny.  So definitely doable.  https://www.ccoderun.ca/programming/ml/cancer_cells.html

**Comment by WilburZjh** [comment]
Hi @stephanecharette , thanks for the response, I have checked this tiny model. Sorry for closing the problem by mistake. My question is how to train a model which an object's class is not a "definitely type A" but a "60% type A"? I did not see a similar solution in this tiny model.

Is there any other demo you have met before based on my situation?

Best

**Comment by stephanecharette** [comment]
Does that happen often?  If so, then you'd need another class, which could be called "looks_like_A_or_B".

**Comment by WilburZjh** [comment]
Yeah, some of the cells are hard to be labeled even for the experts. Therefore, they recorded some labels as "60% confidence for type A, 90% confidence for type B, 70% confidence for type C". That's the reason why I want to ask this question.

**Comment by stephanecharette** [comment]
If they're 90% sure, I'd mark it as such.  But if the % is about equal and you have enough of those that you can use to train, then I'd create a class called "uncertain" or "to-be-reviewed", or whatever is appropriate.

If this only shows up in a few images (not enough to train) then I'd take those images out of the training.
